schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html

```diff
@@ -0,0 +1,46 @@
+<div ng-if="!vm.auth.isAuthenticated()">
+    <div class="card">
+        <div class="card-header">
+            <h2>Authentication
+                <small>Sign in or Create Account to continue with your checkout</small>
+            </h2>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-md-6">
+            <div class="card">
+                <div class="card-header">
+                    <h2>Sign In
+                        <small>Sign in with already created Jasify account</small>
+                    </h2>
+                </div>
+                <div class="card-body card-padding-sm">
+                    <div ng-controller="SignInController as vm"
+                         ng-include="'authenticate/sign-in.html'"></div>
+                </div>
+            </div>
+        </div>
+        <div class="col-md-6">
+            <div class="card">
+                <div class="card-header">
+                    <h2>Create Account
+                        <small>If you don't have Jasify account, create it</small>
```

Comment:
Have a Jasify account

Without the "a" it sounds like a foreigner wrote it

Sent from my iPhone

> On 28.06.2015, at 17:53, "Christian Asmussen" <notifications@github.com> wrote:
> 
> In schedule/schedule-appengine/src/main/client/jasify/checkout-window/checkout-window-anonymous.html:
> 
> > +                <div class="card-header">
> > +                    <h2>Sign In
> > +                        <small>Sign in with already created Jasify account</small>
> > +                    </h2>
> > +                </div>
> > +                <div class="card-body card-padding-sm">
> > +                    <div ng-controller="SignInController as vm"
> > +                         ng-include="'authenticate/sign-in.html'"></div>
> > +                </div>
> > +            </div>
> > +        </div>
> > +        <div class="col-md-6">
> > +            <div class="card">
> > +                <div class="card-header">
> > +                    <h2>Create Account
> > +                        <small>If you don't have Jasify account, create it</small>
> yep #558
> 
> —
> Reply to this email directly or view it on GitHub.
>